improvement: webext `permissions.(request|remove)`: clearer resolve value desciption (#17516)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/permissions/remove/index.md

### Return value

> A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that will be fulfilled with `true` if the permissions listed in the `permissions` argument are now not granted, or `false` otherwise.

Reviewer: ```suggestion
A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that is fulfilled with `true` if the permissions listed in the `permissions` argument are now not granted to the extension, or `false` otherwise.
```
